docs/clear-containers-3.0-quick-install.md

```diff
@@ -0,0 +1,158 @@
+# Clear Containers 3.0 - Quick-Install
+
+## Introduction
+
+### Requirements
+
+  * [go 1.8.3](https://golang.org/)
+  * [glibc-static](https://www.gnu.org/software/libc/libc.html)
+
+### Clear Containers 3.0 components
+
+  * [Runtime](https://github.com/clearcontainers/runtime)
+  * [Proxy](https://github.com/clearcontainers/proxy)
+  * [Shim](https://github.com/clearcontainers/shim)
+  * [Virtcontainers](https://github.com/containers/virtcontainers) (**pause** binary only)
+
+**IMPORTANT:** Do not combine [Clear Containers 2.1](https://github.com/01org/cc-oci-runtime) and [Clear Containers 3.0](https://github.com/clearcontainers) Both projects ship ``cc-proxy`` and they are not compatible with each other.
+
+## Set Environment
+
+### Define GOPATH
+
+```bash
+$ export GOPATH=$HOME/go
+```
+
+### Create GOPATH Directory
+
+```bash
+$ mkdir -p $GOPATH
+```
+
+### Get the code
+
+```bash
+$ go get github.com/clearcontainers/runtime
+$ go get github.com/clearcontainers/proxy
+$ git clone https://github.com/clearcontainers/shim $GOPATH/src/github.com/clearcontainers
+```
+
+## Build and Install Components
+
+### Proxy
+
+```bash
+$ cd $GOPATH/src/github.com/clearcontainers/proxy
+$ make
+     GOBUILD  cc-proxy
+     GEN      cc-proxy.service
+     GEN      cc-proxy.socket
+$ sudo make install
+install -D cc-proxy /usr/libexec/clear-containers/cc-proxy || exit 1;
+install -D -m 644 cc-proxy.service /usr/lib/systemd/system/cc-proxy.service || exit 1;
+install -D -m 644 cc-proxy.socket /usr/lib/systemd/system/cc-proxy.socket || exit 1;
+```
+
+### Shim
+
+```bash
+$ cd $GOPATH/src/github.com/clearcontainers/shim
+$ ./autogen.sh
+$ make
+(CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /home/foo/go/shim/missing autoheader)
```

Comment:
Again, this command generates quite a lot of output which doesn't add anything to the document in my opinion.